src/main/java/com/dumbdogdiner/stickyapi/bukkit/item/generator/PlayerHeadGenerator.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2020 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.bukkit.item.generator;
+
+import com.destroystokyo.paper.profile.PlayerProfile;
+import com.destroystokyo.paper.profile.ProfileProperty;
+import com.dumbdogdiner.stickyapi.bukkit.user.StickyUserBukkit;
+import org.bukkit.Bukkit;
+import org.bukkit.Material;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.SkullMeta;
+import com.google.gson.Gson;
+
+import java.net.URL;
+import java.util.Base64;
+import java.util.UUID;
+
+public class PlayerHeadGenerator {
+    private SkullMeta meta = (SkullMeta) (new ItemStack(Material.PLAYER_HEAD, 1)).getItemMeta();
+
+    PlayerProfile ownerProfile;
+
+    public PlayerHeadGenerator(UUID playerId){
```

Comment:
```suggestion
    public PlayerHeadGenerator(@NotNull UUID playerId) {
```

Use annotations please, this is for Kotlin support.